examples/apiary_scenarios/asset_accept_workflow.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * This file is part of the Ingram Micro Cloud Blue Connect SDK.
+ *
+ * @copyright (c) 2020. Ingram Micro. All Rights Reserved.
+ */
+const { ConnectClient, Fulfillment } = require('@cloudblueconnect/connect-javascript-sdk');
+const fetch = require('node-fetch');
+const config = require('./config.json');
+
+/* eslint-disable no-console */
+
+/**
+ * This module is an example of fulfillment flow between
+ * Cloudblue Connect and Vendor System.
+ * This connect with the Vendor System using an
+ * example of API Vendor System implemented in apiary.io
+ * and documented in the Cloudblue Connect Documentation Portal:
+ * https://connect.cloudblue.com/community/sdk/vendor-scenario-example/fulfillment/
+ *
+ * This module accept the Purchase Request into Connect when the tenant is ready in Vendor System.
+ *
+ * @category Operations
+ */
+const client = new ConnectClient(
+  config.url,
+  config.apiKey,
+);
+const fulfillment = new Fulfillment(client);
+const urlBase = 'https://SET_YOUR_OWN_SAMPLE.apiary-mock.com/';
+let tenantIdParam;
+let templateIdParam;
+
+/**
+ * Process Purchase Request into Cloudblue Connect
+ *
+ * @param   {object}  element  The request of Cloudblue Connect.
+ *
+ */
+function processRequest(element) {
+  if (element.type === 'purchase') {
+    element.asset.params.forEach((param) => {
+      if (param.id === 'tenantId') {
+        tenantIdParam = param.value;
+      }
+    });
+    if (tenantIdParam !== '') {
+      element.asset.configuration.params.forEach((param) => {
+        if (param.id === 'templateId') {
+          templateIdParam = param.value;
+        }
+      });
+      const body = {
+        template_id: templateIdParam,
+      };
+      const approveRequests = async () => {
+        const responseApprove = await fulfillment.approveRequest(element.id, body);
+        return responseApprove;
+      };
+      approveRequests()
+        .then((responseApprove) => console.log(responseApprove))
+        .catch((e) => console.log(e));
+    } else {
+      throw Error('This Tenant not exist in vendor');
```

Comment:
throw new Error (it is a class it must be instantiated with the new operator).

Reply:
Fixed.